draft-sharabayko-mops-srt.md

```diff
@@ -2086,15 +2086,584 @@ and RTTVar are updated by the socket based on algorithms for the sender
+detected via loss reports, the sending rate is gradually increased.
+On the other hand, if a network congestion is detected, the algorithm
+slows down and decreases the sending rate to reduce following packet
+loss. The slow start phase runs exactly once at the beginning of a
+connection and stops when a packet loss occurs, congestion window
+size reaches its maximum value, or on a timeout event.
+
+The detailed algorithm behaviour at both phases is described
+in {{default-fileCC-slow-start}} and
+{{default-fileCC-congestion-avoidance}}, respectively.
+
+#### Slow Start {#default-fileCC-slow-start}
+
+During the slow start phase, the packet sending period PKT_SND_PERIOD
+is kept at 1 microsecond in order to send packets as fast as possible,
+but not with an infinite rate.
+The initial value of congestion window size CWND_SIZE is set to 16
+packets. CWND_SIZE has an upper threshold, the maximum allowed
+congestion window size MAX_CWND_SIZE, which is used for safety purposes
+(even if there is no packet loss, the slow start phase has to stop
+at a certain point). The threshold can be set to the maximum of
+the receiver buffer size (12 MB).
+
+On an acknowledgement (ACK) packet reception:
+
+Step 1. If the time passed since the last time LastRCTime the sending rate
+was either increased, or kept, is less than RC_INTERVAL:
+
+a. Keep the sending rate at the same level;
+
+b. Stop.
+
+    <CODE BEGINS>
+    if (currTime - LastRCTime < RC_INTERVAL)
+    {
+        Keep the sending rate at the same level;
+        Stop;
+    }
+    <CODE ENDS>
+
+where currTime is the current time, in microseconds; LastRCTime is
+the last time the sending rate was either increased, or kept,
+in microseconds.
+
+Step 2. Update the value of LastRCTime to the current time:
+
+~~~
+LastRCTime = currTime
+~~~
+
+Step 3. The size of congestion window CWND_SIZE is increased by the
+difference in sequence numbers of the data packet being acknowledged
+ACK_SEQNO and the last acknowledged data packet LAST_ACK_SEQNO:
+
+~~~
+CWND_SIZE += ACK_SEQNO - LAST_ACK_SEQNO
+~~~
+
+Step 4. The sequence number of the last acknowledged data packet
+LAST_ACK_SEQNO is updated as follows:
+
+~~~
+LAST_ACK_SEQNO = ACK_SEQNO
+~~~
+
+Step 5. If calculated at Step 3 congestion window size CWND_SIZE is greater
+than the upper threshold MAX_CWND_SIZE, slow start phase ends.
+Set the packet sending period PKT_SND_PERIOD as follows:
+
+    <CODE BEGINS>
+    if (RECEIVING_RATE > 0)
+        PKT_SND_PERIOD = 1000000 / RECEIVING_RATE;
+    else
+        PKT_SND_PERIOD = CWND_SIZE / (RTT + RC_INTERVAL);
+    <CODE ENDS>
+
+where
+
+- RECEIVING_RATE is the rate at which packets are being received,
+in packets per second, reported by the receiver and smoothed at
+the sender side (see {{ctrl-pkt-ack}}, {{bandwidth-estimation}});
+- RTT is the round-trip time estimation, in microseconds,
+reported by the receiver and smoothed at the sender side (see
```

Comment:
smoothed?

Reply:
the same explanation as for the very first smoothed RTT